custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_v2.json

```diff
@@ -1377,6 +1377,43 @@
               }
             },
             "property_value": 183
+          },
+          {
+            "type": "or",
```

Comment:
@calellowitz 

Shouldn't this be an "and" (the person is not not_registered AND not migrated)?